Cubical/Categories/Displayed/Limits/Terminal.agda

```diff
@@ -0,0 +1,71 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Displayed.Limits.Terminal where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Equiv
+
+open import Cubical.Data.Sigma
+open import Cubical.Data.Unit
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Presheaf
+open import Cubical.Categories.Instances.Sets
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Displayed.Presheaf
+open import Cubical.Categories.Displayed.Functor
+open import Cubical.Categories.Limits.Terminal
+open import Cubical.Categories.Limits.Terminal.More
+open import Cubical.Categories.Presheaf.More
+
+-- There are multiple definitions of terminal object in a displayed category:
+-- 1. A terminal object in the total category, which is preserved by projection
+-- 2. A terminal object in the *fiber* of an object
```

Comment:
Being a lil shaky on how the limits below get instantiated, is 2. equivalent to this concrete description of a terminal object in a fiber? Reading Jacob's book asap

For `1 `your displayed terminal object over some `c`, for all `d : ob[c]`, `Hom[id][d, 1]` is contractible. If this is true, can you swap `id` out for any other endomorphism?